app/static/update-browser.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
```

Comment:
```suggestion
<html lang="en">
```
suggested by sonarcloud